restricted_QAQC/global.R

```diff
@@ -210,33 +219,51 @@ con = dbConnect(y,
                 dbname = dbname)
 tables = dbListTables(con)
 
-# # Code for one-time use: to load data into mysql
-# dataCurrent <- read.csv("data/current_clean20181202.csv", stringsAsFactors = FALSE, na.strings=c(""," ", "NA"))
+# Code for one-time use: to load data into mysql
+# dataCurrent <- read.csv("data/current_clean.csv", stringsAsFactors = FALSE, na.strings=c(""," ", "NA"))
```

Comment:
these files are very obsolete, even the one called "current_clean.csv". your database should have been set up correctly after we built it together, but it's possible that it has duplications or other weirdness if you ran `dbWriteTable` below.

Sorry, I haven't noticed those lines since 2018! You can rebuild your database from `hbef_backup_1d.gz` again like we did ~March 1.